packages/react/src/dialog/popup/cssVars.ts

```diff
@@ -0,0 +1,11 @@
+import { generateCssVariables } from '../../utils/generateCssVariables';
+
+export type DialogPopupCssVars = {
+  /**
+   * Indicates how many dialogs are nested within.
+   * @type number
+   * */
+  nestedDialogs: string;
+};
+
+export const cssVars: DialogPopupCssVars = generateCssVariables(['nestedDialogs']);
```

Comment:
I added this util similar to the utility for generating classes we have in other projects: https://github.com/mui/material-ui/blob/master/packages/mui-material/src/Dialog/dialogClasses.ts#L43, to make sure we don't have typos while generating the CSS variables.